lib/clearance.rb

```diff
@@ -10,7 +10,4 @@
 require 'clearance/constraints'
 
 module Clearance
```

Comment:
I was thinking it'd be a good place to add any top-level RDoc, but maybe I'm wrong. I've never RDoc'd anything.